modules/delegates/components/DelegateVoteHistory.tsx

```diff
@@ -0,0 +1,20 @@
+import { PollVoteHistoryList } from 'modules/polls/components/PollVoteHistoryList';
+import { AddressAPIStats } from 'modules/address/types/addressApiResponse';
+import { Box, Divider, Text } from 'theme-ui';
+import { Delegate } from '../types';
+
+export function DelegateVoteHistory({ delegate, stats }: { delegate: Delegate, stats: AddressAPIStats}): React.ReactElement {
+  return (
+    <Box p={[3,4]}>
+      <Text as="p" sx={{
+        fontSize: '18px',
```

Comment:
In general, I think we should stick to our font-size variables from our theme spec.

Reply:
@adamgoth  do you know how to get a 18px font-size with our theme spec?

Reply:
This has come up before. I think @b-pmcg pointed this out not too long ago, but if 18px isn't available as a size in our spec, then maybe the question should be why are we using it in the design? That means it's not appearing anywhere else in the app. Maybe we should check with Tiago on a different size to use. Or just use your best judgment between what's available in the spec.